experimental/cloudstrapper/playbooks/roles/ansible-server/tasks/main.yaml

```diff
@@ -0,0 +1,16 @@
+#Installing Ansible Server
+#Needs to be run as sudo user (instead of becoming sudo user for each play)
+
+
+- name: installing ansible dependent packages
+  apt: 
+    name: "{{ bootPackages }}" 
```

Comment:
shouldn't this be {{item}}

Reply:
Conversely, I need to remove the with_items since the bootPackages variable has a growing list of new packages to be installed. Made the change.